packages/commons/src/config-types/workspace-definition/__test__/substituteEnvVariables.test.ts

```diff
@@ -0,0 +1,18 @@
+import yaml from "js-yaml";
+import { substituteEnvVariables } from "../substituteEnvVariables";
+
+describe("substituteEnvVariables", () => {
+    it("basic", () => {
+        process.env.ENV_VAR = "test";
```

Comment:
Nbd, but this will pollute the actual environment, you can use jest.mock in general for this kind of thing

Reply:
got it, i googled on stack overflow and this was the most upvoted answer